practice/src/index.html

```diff
@@ -6,8 +6,47 @@
     <link rel="stylesheet" href="./css/main.css" />
     <title>Products Management</title>
   </head>
-
   <body>
+    <!-- Card 1-->
+    <div class="card card-product">
+      <div class="card-header">
+        <svg width="15" height="15">
+          <use xlink:href="./assets/images/icons/icons.svg#delete-icon" />
+        </svg>
+      </div>
+      <div>
+        <figure class="card-image">
+          <img src="#" alt="food image" />
+        </figure>
+        <div class="card-title">
+          <p class="main-title card-name">Spicy seasoned seafood noodles</p>
+          <div class="card-desc main-desc">
```

Comment:
Should define the common class first, `<div class="main-desc card-desc">`

Reply:
Please douple check

Reply:
I have fixed it